package.json

```diff
@@ -1,14 +1,14 @@
 {
 	"name": "extron-smx",
-	"version": "1.0.1",
+	"version": "1.0.2",
 	"api_version": "1.0.0",
 	"keywords": [
 		"Protocol",
 		"Matrix"
 	],
 	"manufacturer": "Extron",
 	"product": "SMX Matrix",
-	"shortname": "Extron SMX Matrix",
+	"shortname": "SMX Matrix",
```

Comment:
shortname should be without spaces, preferrably only small letters, and if possible, even shorter. Maybe only smx? (I know it is not documented anywhere 😞 sorry for that)

Reply:
I'd copied this from the DXP module and was trying to shorten so it looked pretty in the GUI!